veashi-contracts/.env.example

```
@@ -35,3 +35,9 @@ CCIP_ADAPTER_ROUTER=0x0BF3dE8c5D3e8A2B34D2BEeB17ABfCeBaf363A59
 CCIP_REPORTER_ROUTER=0x2a9C5afB0d0e4BAb2BCdaE109EC4b0c4Be15a165
 CCIP_REPORTER_CHAIN_SELECTOR=3478487238524512106
 CCIP_ADAPTER_CHAIN_SELECTOR=16015286601757825753
+
+# DeBridge
+DEBRIDGE_REPORTER_GATE_ADDRESS=0x43dE2d77BF8027e25dBD179B491e8d64f38398aA
+DEBRIDGE_ADAPTER_GATE_ADDRESS=0x43dE2d77BF8027e25dBD179B491e8d64f38398aA
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEBRIDGE_ADAPTER_GATE_ADDRESS key should go before the DEBRIDGE_REPORTER_GATE_ADDRESS key

(UnorderedKey)